### PR TITLE
fix validate for all langs

### PR DESCRIPTION
**Pull request by mkangia**
Allows user to validate translations for all languages.

Feature flag: validate_app_translations

https://dimagi-dev.atlassian.net/browse/ICDS-508

### CODE REVIEW

File: corehq/apps/translations/validator.py [hunk 1]
```diff
@@ -36,7 +36,9 @@ def __init__(self, app, uploaded_workbook, lang_to_compare, lang_prefix='default
         self.expected_rows = None
         self.lang_prefix = lang_prefix
         self.lang_cols_to_compare = [self.lang_prefix + self.app.default_language]
-        if lang_to_compare != self.app.default_language:
+        if lang_to_compare not in app.langs:
```
Comment (orangejenny) [inline corehq/apps/translations/validator.py]: Setting this to None is a bit confusing. I think this is equivalent and easier to read:
```
if lang_to_compare in app.langs and lang_to_compare != self.app.default_language:
    self.lang_cols_to_compare.append(self.lang_prefix + lang_to_compare)
self.default_language_column = self.lang_prefix + self.app.default_language
target_lang = lang_to_compare if lang_to_compare in app.langs else self.app.default_language
```

**Review by orangejenny — APPROVED** [review]

**Comment by orangejenny** [comment]
Are there tests for this?

**Comment by mkangia** [comment]
@orangejenny  Nope, there are not.

File: corehq/apps/translations/validator.py (hunk 1 above)
Comment (mkangia) [reply corehq/apps/translations/validator.py]: Thank you @orangejenny. Updated

**Review by orangejenny — APPROVED** [review]

**Comment by orangejenny** [comment]
So, how much effort would it take to add a test?

(You knew I was going to ask that...)

**Comment by mkangia** [comment]
It will be some effort @orangejenny . I can take that in a follow up PR. There are quite some tests around this functionality that can be added.

**Comment by orangejenny** [comment]
Cool - agreed it doesn't need to block this PR, I'd just like to get better test coverage in app manager over time.

**Pull request merged**